reactnative/RNPlayground/app/screens/AboutScreen.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC } from "react"
+import { observer } from "mobx-react-lite"
+import { StackScreenProps } from "@react-navigation/stack"
+import { AppStackScreenProps } from "../navigators"
+import { Image, ImageStyle, TextStyle, View, ViewStyle } from "react-native"
+import {
+  Text, Screen
+} from "../components"
+import { isRTL } from "../i18n"
+import { colors, spacing } from "../theme"
+import { useSafeAreaInsetsStyle } from "../utils/useSafeAreaInsetsStyle"
+
+export const AboutScreen: FC<StackScreenProps<AppStackScreenProps, "About">> = observer(function AboutScreen() {
+    const $bottomContainerInsets = useSafeAreaInsetsStyle(["bottom"])
+
+    return (
+      <View style={$container}>
+        <View style={$topContainer}>
+          <Text
+            testID="welcome-heading"
+            style={$welcomeHeading}
+            tx="aboutScreen.readyForLaunch"
+            preset="heading"
+          />
+          <Text tx="aboutScreen.postscript" preset="subheading" />
+        </View>
+
+        <View style={[$bottomContainer, $bottomContainerInsets]}>
+          <Text tx="aboutScreen.exciting" size="md" />
+        </View>
+      </View>
+    )
+})
+
+const $container: ViewStyle = {
+  flex: 1,
+  backgroundColor: colors.background,
+}
+
+const $topContainer: ViewStyle = {
```

Comment:
btw the elements of the style looks like very similar to the ones on regular react app

Reply:
Thats because they are! This PR was mostly for the aim of me working on the Nav, not particularly the UI.